src/Reflection/Exception/InvalidNode.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BetterReflection\Reflection\Exception;
+
+use RuntimeException;
+
+class InvalidNode extends RuntimeException
+{
+    public static function create() : self
+    {
+        return new self('Invalid node');
```

Comment:
This exception is indeed too simplistic. I would suggest building a pretty-printer and dumping the code in here

Reply:
Done.